### PR TITLE
Request 123 - "praca domowa"

### PR DESCRIPTION
**Pull request by MichalLuczak143**
Request 123 - "praca domowa" zgodnie ze wczorajszym callem

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -4,18 +4,23 @@
 def run_app(prices):
 
     # very important things
-    time.sleep(60.0)
+    time.sleep(02.00)
```
Comment (lukaszszymankiewicz) [inline app.py:7]: please, revert the change here. app does not work as before :)

File: tests/test_src.py [hunk 2]
```diff
@@ -1,25 +1,21 @@
 from app import set_prices
-from app import app
+
 
 def test_set_prices_sorting_works():
-    #
+    # Sortowanie
```
Comment (lukaszszymankiewicz) [inline tests/test_src.py:5]: please, keep the comments in English, I cannot understand it now:)

File: tests/test_src.py [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
     prices = [10, 30, 20]
     prices.sort()
-   # return prices
 
-    #
+    # Sorotwanie z mnożnikiem
```
Comment (lukaszszymankiewicz) [inline tests/test_src.py:9]: ditto

File: tests/test_src.py [hunk 4]
```diff
@@ -1,25 +1,21 @@
...
     result = set_prices(prices, 1.0)
 
-
-    # if result_sorted != result:
-    #     raise AssertionError()
+    # Sprawdzenie czy działa
```
Comment (lukaszszymankiewicz) [inline tests/test_src.py:12]: ditto

**Review by lukaszszymankiewicz — CHANGES_REQUESTED** [review]

File: tests/test_src.py [hunk 5]
```diff
@@ -1,25 +1,21 @@
...
     assert prices == result
     assert prices == [10, 20, 30]
 
 
 def test_set_prices_multiplying_works(prices=[10, 20, 30], a=2):
```
Comment (lukaszszymankiewicz) [inline tests/test_src.py:18]: I think that 'prices' and 'a' should belong to unit-test code not parameters. Please move these two below

File: tests/test_src.py (hunk 2 above)
Comment (MichalLuczak143) [reply tests/test_src.py:5]: learn polish nub. Polska górom

File: tests/test_src.py (hunk 3 above)
Comment (MichalLuczak143) [reply tests/test_src.py:9]: images

**Comment by lukaszszymankiewicz** [comment]
Hi @MichalLuczak143, please do not close the Pull Request just like that :). Closing it, like that, will not apply any of your changes to the codebase.

Please align the code with the comments, commit and push the changes. Then, after my another review I will resolve my comments (if I will agree with the changes), and then (and only then), we could have this PR merged.

The aim of this exercise was to see the full flow of git VCS involved project.

If you have any question. please do not hesitate to ask.

File: app.py (hunk 1 above)
Comment (lukaszszymankiewicz) [reply app.py:7]: still waiting :)

File: tests/test_src.py (hunk 5 above)
Comment (lukaszszymankiewicz) [reply tests/test_src.py:18]: still waiting :)

**Review by lukaszszymankiewicz — COMMENTED** [review]
still waiting:)

**Comment by MichalLuczak143** [comment]
I had a lot work, please don't fire me Mr. Szymankiewicz. I have Buła to feed. Let's check it on call soon

**Comment by MichalLuczak143** [comment]
I had a lot work, please don't fire me Mr. Szymankiewicz. I have Buła to feed. Let's check it on call soon
